web/src/main/java/com/futurewei/alcor/web/entity/dataplane/InternalPortEntity.java

```diff
@@ -27,8 +27,8 @@
     @JsonProperty("routes")
     private List<RouteEntity> routes;
 
-    @JsonProperty("neighbor_info")
-    private List<NeighborInfo> neighborInfos;
+    @JsonProperty("neighbor_ids")
```

Comment:
This is aligned with the DPM-ACA contract change 👍

Reply:
For reference, the latest DPM-ACA contract is here: https://github.com/er1cthe0ne/alcor/blob/da2021a443444046a122f6b06c3efaad70087f2e/schema/proto3/neighbor.proto

Reply:
I think InternalPortEntity don't need neighborInfos, has updated by the new commit.

Reply:
I have changed InternalPortEntity in bcea6b186c0df1ecc4fef87d74c2b87daec03639 per discussion with team

Reply:
@chenpiaoping could you add more details in PR's desc section?